modules/telestion-api/src/main/java/de/wuespace/telestion/api/message/HeaderInformation.java

```diff
@@ -0,0 +1,823 @@
+ * <p>
+ * There are numerous interfaces that simplifies the usage with the existing features of Vert.x
+ * and better support through the {@link de.wuespace.telestion.api.verticle.trait.WithEventBus WithEventBus trait}
+ * for {@link io.vertx.core.Verticle Vert.x verticles}
+ * and especially the {@link de.wuespace.telestion.api.verticle.TelestionVerticle TelestionVerticle}.
+ *
+ * <h2>Usage</h2>
+ * <pre>
+ * {@code
+ * public class MyVerticle extends TelestionVerticle<GenericConfiguration> implements WithEventBus {
+ *     @Override
+ *     public void onStart() {
+ *         var requestInfos = new HeaderInformation()
+ *                 .add("version", 1)
+ *                 .add("log-enabled", true)
+ *                 .add("my-initial-char", 'C').
+ *                 .add("my-coworkers-name", "fussel178");
+ *
+ *         request("request-address", "Hello with Info", requestInfos).onSuccess(message -> {
+ *             var responseInfos = HeaderInformation.from(message);
+ *
+ *             logger.info("Received version: {}", responseInfos.getInt("version", -1));
+ *             logger.info("Received log enabled state: {}", responseInfos.getBoolean("log-enabled", false));
+ *             logger.info("Received initial char: {}", responseInfos.getChar("my-initial-char", '\0'));
+ *             logger.info("Received coworker name: {}", responseInfos.get("my-coworkers-name", "unset"));
+ *         });
+ *     }
+ * }
+ * }
+ * </pre>
+ *
+ * @author Cedric Boes (@cb0s), Ludwig Richter (@fussel178)
+ */
+public class HeaderInformation {
+
+	/**
+	 * Merges an array of header information into one header information object.
+	 *
+	 * @param information multiple header information that should be merged into one
+	 * @return the merged header information
+	 */
+	public static HeaderInformation merge(HeaderInformation... information) {
+		return merge(Arrays.stream(information));
+	}
+
+	/**
+	 * Merges a list of header information into one header information object.
+	 *
+	 * @param list multiple header information that should be merged into one
+	 * @return the merged header information
+	 */
+	public static HeaderInformation merge(List<HeaderInformation> list) {
+		return merge(list.stream());
+	}
+
+	/**
+	 * Merges a stream of header information into one header information object.
+	 *
+	 * @param stream a stream that contains multiple header information that should be merged into one
+	 * @return the merged header information
+	 */
+	public static HeaderInformation merge(Stream<HeaderInformation> stream) {
+		var headers = stream.map(HeaderInformation::getHeaders);
+		return new HeaderInformation(MultiMapUtils.merge(headers));
+	}
+
+	/**
+	 * Creates new header information from existing {@link MultiMap Vert.x headers}.
+	 *
+	 * @param headers the basic {@link MultiMap Vert.x headers} that should be wrapped into the header information
+	 * @return new header information with the wrapped {@link MultiMap Vert.x headers}
```

Comment:
```suggestion
	 * @return new header information object with the wrapped {@link MultiMap Vert.x headers}
```

Reply:
Done in [`2cd29d6` (#484)](https://github.com/wuespace/telestion-core/pull/484/commits/2cd29d6f1426056b991dcbdcda1b3d2c3274fe9e)